content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+
+## Before you begin
+
+In addition to the prerequisites for installing Istio the following setup is required for this example:
+
+*  This sample requires a valid Google Cloud Platform project with billing enabled. If you are
+   not an existing GCP user, you may be able to enroll for a $300 US [Free Trial](https://cloud.google.com/free/) credit.
+
+   *  [Create a Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) to
+      host your GKE clusters.
+
+*  Install and initialize the [Google Cloud SDK](https://cloud.google.com/sdk/install)
+
+## Create the GKE Clusters
+
+1.  Set the default project for `gcloud` to perform actions on:
+
+    {{< text bash >}}
+    $ gcloud config set project myProject
+    $ proj=$(gcloud config list --format='value(core.project)')
+    {{< /text >}}
+
+1.  Create 2 GKE clusters for use with the multicluster feature.  _Note:_ `--enable-ip-alias` is required to
+    allow inter-cluster direct pod-to-pod communication.  The `zone` value must be one of the
+    [GCP zones](https://cloud.google.com/compute/docs/regions-zones/).
+
+    {{< text bash >}}
+    $ zone="us-east1-b"
+    $ cluster="cluster-1"
+    $ gcloud container clusters create $cluster --zone $zone --username "admin" \
+      --cluster-version "1.9.6-gke.1" --machine-type "n1-standard-2" --image-type "COS" --disk-size "100" \
+      --scopes "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.read_only",\
+      "https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring",\
+      "https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly",\
+      "https://www.googleapis.com/auth/trace.append" \
+      --num-nodes "4" --network "default" --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --async
+    $ cluster="cluster-2"
+    $ gcloud container clusters create $cluster --zone $zone --username "admin" \
+      --cluster-version "1.9.6-gke.1" --machine-type "n1-standard-2" --image-type "COS" --disk-size "100" \
+      --scopes "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.read_only",\
+      "https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring",\
+      "https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly",\
+      "https://www.googleapis.com/auth/trace.append" \
+      --num-nodes "4" --network "default" --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --async
+    {{< /text >}}
+
+1.  Wait for clusters to transition to `RUNNING` state:
+
+    {{< text bash >}}
+    $ gcloud container clusters list
+    {{< /text >}}
+
+1.  Get the clusters' credentials (by default stored in `~/.kube/config` or in the file the KUBECONFIG environment
```

Comment:
Here and elsewhere, add colons in the sentence before a text block

Reply:
ok